keps/sig-windows/1122.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 1122
```

Comment:
this file is in the wrong directory. it should be `keps/prod-readiness/sig-windows/1122.yaml`

Reply:
fixed it. Thank you!